testsuites/listener/shared/client_sg/test_design_docs_update.py

```diff
@@ -82,3 +82,98 @@ def test_design_doc_update(setup_client_syncgateway_test):
 
     content_view_rows_2 = client.get_view(url=ls_url, db=ls_db, design_doc_name=d_doc_name, view_name="content_view")
     client.verify_view_row_num(view_response=content_view_rows_2, expected_num_rows=5)
+
+
+@pytest.mark.sanity
```

Comment:
do you want to mark this as sanity? I guess it is not required

Reply:
removed.